test/WebsocketTest.php

```diff
@@ -388,7 +388,7 @@ function onData(int $clientId, Websocket\Message $msg) {
     function testFragmentation() {
         Loop::execute(\Amp\wrap(function () {
             list($endpoint, $client, $sock, $server) = yield from $this->initEndpoint(new NullWebsocket);
-            $endpoint->send(null, str_repeat("*", 131046), true)->when(function() use ($sock, $server) { stream_socket_shutdown($sock, STREAM_SHUT_WR); $server->requireClientFree = true; });
+            $endpoint->broadcast(null, str_repeat("*", 131046), true)->when(function() use ($sock, $server) { stream_socket_shutdown($sock, STREAM_SHUT_WR); $server->requireClientFree = true; });
```

Comment:
Here `$endpoint` is actually an instance of `Rfc6455Gateway`. I didn't change the argument order of that internal class or the naming in the tests. I'll update both.